examples/gno.land/r/faucet/admin.gno

```diff
@@ -5,44 +5,90 @@ import (
 	"std"
 )
 
-func AdminSetInPause(inPause bool) error {
+func AdminSetInPause(inPause bool) string {
```

Comment:
please roll back to `error`, I'll manage to make the `error` type transferred and readable from `gnokey`.

Reply:
I think only the basic types are serialized.

Reply:
1) Only basic types are supported for parameters and return values in public API. Will only gnokey be able to read the error message if we introduce an error type in the returned public API? 

2) I have completed integration testing with the bot, and the faucet is ready to use. Can we update it later when the return error type is supported?

Reply:
1) Only basic types are supported for parameters and return values in public API. Will only gnokey be able to read the error message if we introduce an error type in the returned public API? 

2) I have completed integration testing with the bot, and the faucet is ready to use. Can we update it later when the return error type is supported?

Reply:
Yes, sure, let's do string for now, and error later :+1: